### DISCUSSION TITLE
quick question: is postgresl support available in EE only?

### DISCUSSION THREAD

**Discussion by missedone** [mend/renovate-ce-ee#428] [Q&A] [answered]
https://github.com/mend/renovate-ce-ee/releases/tag/6.9.0

> feat: Add support for PostgreSQL database
enabled by setting MEND_RNV_DATA_HANDLER_TYPE=postgresql and native PostgreSQL env variables on the server container (PGDATABASE, PGUSER, PGPASSWORD, PGPORT, PGHOST)

i see the env vars are in the EE charts only, so wondering if the feature is available in EE only.
also what's the permission needed by the DB user? i assume it need to be able to create schema and tables?

**Comment by rarkins** [comment, marked as answer]
CE supports postgres. The chart will be updated later

The user needs to be able create schemas/tables

**Comment by missedone** [comment]
that's great, can't wait for it, looking forward to the CE helm chart update, thx
